src/app/i18n.ts

```diff
@@ -89,7 +89,7 @@ export function initi18n(): Promise<unknown> {
       {
         initImmediate: true,
         compatibilityJSON: 'v3',
-        debug: $DIM_FLAVOR === 'dev',
+        debug: false,
```

Comment:
Do we want this change? Or was it a change while debugging?

Reply:
Yes, I don't think it's necessary.